docs/ephemeral-testing.md

```diff
@@ -0,0 +1,46 @@
+# Testing Changes in Ephemeral
+
+tl;dr
+1) Make your changes
+2) Build and push you changes in a container image: `export IMAGE=quay.io/your-quay-repo/kessel-relations-api make docker-build-push`
```

Comment:
I know we have 2 implementations to build and push changes depending on whether you're on mac or not, I feel like we could link to that section here. WDYT?

Reply:
Where is the mac one, this command is complaining? :)
Get this for zsh:
```
export: not valid in this context: docker-build-push
```
Also complains in bash.

Also (since I seem to have exported the var to my env), just doing `make docker-build-push` complains with 
```
QUAY_USER and QUAY_TOKEN must be set
```
which is totally fair, but maybe we should add something in about that. *goes off to poke at quay*

Reply:
Ahh yep, ill make sure to add a link for Mac (already in [README ](https://github.com/project-kessel/relations-api/blob/main/README.md#build-container-images-macos))
Ill split up the export commands to ensure it functions ok, the QUAY_USER and QUAY_TOKEN files require credentials to login to Quay. These can be your own Quay credentenials (QUAY_TOKEN just being the password you login to Quay with). Its so it can push after you build which you can't push to quay without logging in with podman/docker in general.